llvm/include/llvm/IR/RuntimeLibcalls.td

```diff
@@ -1100,19 +1122,25 @@ defvar DefaultRuntimeLibcallImpls_f128 =
     !filter(entry, AllDefaultRuntimeLibcallImpls,
             !match(!cast<string>(entry.Provides), "_F128"));
 
-defvar DefaultRuntimeLibcallImpls =
```

Comment:
Why not the other way around? List seems to be a natural choice. Do you need a tree structure for some reason?

Reply:
With the dag you can do set operations on them. The list versions are much more cumbersome and don't have a full set of operators. It all gets flattened in the final interpretation